stateful-workload-filestore/indexInfo.html

```diff
@@ -22,22 +22,21 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
 
     <script>
-        $(function () {
-            $("#includedContent").load("indexData.html", function(){
-                var rows = $('text').length;
-                document.getElementById('totalRows').innerHTML = rows;
-                if (rows > 0){
-                    document.getElementById('latestDate').innerHTML = $('text')[rows-1].innerHTML;
-                    document.getElementById('latestWriter').innerHTML = $('text2')[rows-1].innerHTML;
+        $(function() {
+            $('#includedContent').load('indexData.html', function() {
+                const rows = $('text').length;
+                $('#totalRows').html(rows);
+                if (rows > 0) {
+                    $('#latestDate').html($('text')[rows - 1].innerHTML);
+                    $('#latestWriter').html($('text2')[rows - 1].innerHTML);
                 }
-                var allWriters = $('text2');
+
+                const allWriters = $('text2');
```

Comment:
**Change 4:** We're using `const` instead of `var` and `let`. I'm just aligning with [a popular philosophy](https://javascript.plainenglish.io/why-const-variables-need-to-be-your-default-choice-in-javascript-b9fb4a592688) in JavaScript that suggests `const` for all variables that don't get reassigned a value. Sorry, this is just me being pendantic. 😅